.github/workflows/system-tests.yml

```diff
@@ -11,9 +11,7 @@ on:
 env:
   REGISTRY: ghcr.io
   REPO: ghcr.io/datadog/dd-trace-rb
-  ST_REF: main
-  FORCE_TESTS:
-  FORCE_TESTS_SCENARIO:
+  ST_REF: main # This must always be set to `main` on dd-trace-rb's master branch
```

Comment:
If we control the naming, I would suggest `SYSTEM_TESTS_REF` or so.

Reply:
Fixed in f7dc462f057c266beba9f44ed93b7d3fc38930a5